geonode/upload/tasks.py

```diff
@@ -173,7 +173,9 @@ def _update_upload_session_state(self, upload_session_id: int, _is_owned: bool =
             session = _upload.get_session.import_session
             if not session or session.state != Upload.STATE_COMPLETE:
                 session = gs_uploader.get_session(_upload.import_id)
-            
+
+            _is_owned = any([_upload.get_session.owned, _is_owned])
```

Comment:
not sure about this:
-  `_is_owned` is True if we are in the threadh that just created the UploadSession, so we can go on
-  `_upload.get_session.owned` means that the session is owned by someone else